src/lib.rs

```diff
@@ -1,88 +1,12 @@
-pub mod common {
-    pub mod adxl345;
-    pub mod analog;
-    pub mod app_client;
-    pub mod base;
-    pub mod board;
-    pub mod camera;
-    pub mod config;
-    pub mod digital_interrupt;
-    pub mod encoder;
-    pub mod entry;
-    pub mod gpio_motor;
-    pub mod grpc;
-    pub mod grpc_client;
-    pub mod i2c;
-    pub mod log;
-    pub mod math_utils;
-    pub mod moisture_sensor;
-    pub mod motor;
-    pub mod movement_sensor;
-    pub mod mpu6050;
-    pub mod registry;
-    pub mod robot;
-    pub mod sensor;
-    pub mod status;
-    pub mod stop;
-    pub mod webrtc {
-        pub mod api;
-        pub mod candidates;
-        pub mod certificate;
-        pub mod dtls;
-        pub mod exec;
-        pub mod grpc;
-        pub mod ice;
-        pub mod io;
-        pub mod sctp;
-    }
-    pub mod conn {
-        pub mod errors;
-        pub mod mdns;
-        pub mod server;
-        mod utils;
-    }
-}
+pub mod common;
```

Comment:
I think this reorganization is definitely more intuitive at the very least, but it might be two big of a change as we're going into beta? We might want to hold off on this

Reply:
Good to know it feels more intuitive. 

While it definitely feels like a big change, it doesn't affect any of the main logic or imports.

There is an alternative though if you'd still like the docs without altering the structure.

https://doc.rust-lang.org/rustdoc/write-documentation/the-doc-attribute.html#the-doc-attribute

This would just be missing the compartmentalizing advantages of having inline documentation.